app/views/host_aggregate/edit.html.haml

```diff
@@ -1,36 +1,3 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "hostAggregateFormController"}
-  = render :partial => "layouts/flash_msg"
-  %h3
-    = _('Edit Host Aggregate')
-  .form-horizontal
-    .form-group
-      %label.col-md-2.control-label
-        = _('Host Aggregate Name')
-      .col-md-8
-        %input.form-control{:type          => "text",
-                            :name          => "name",
-                            'ng-model'     => "hostAggregateModel.name",
-                            'ng-maxlength' => 128,
-                            :miqrequired   => false,
-                            :checkchange   => true}
-  %table{:width => '100%'}
-    %tr
-      %td{:align => 'right'}
-        #buttons_on
-          = button_tag(_("Save"),
-                       :class        => "btn btn-primary",
-                       "ng-click"    => "saveClicked()",
-                       "ng-disabled" => "angularForm.$pristine || angularForm.$invalid",
-                       "ng-class"    => "{ disabled: angularForm.$pristine || angularForm.$invalid}")
-          = button_tag(_("Reset"),
-                       :class        => "btn btn-primary",
-                       "ng-click"    => "resetClicked()",
-                       "ng-disabled" => "angularForm.$pristine",
-                       "ng-class"    => "{ disabled: angularForm.$pristine}")
-          = button_tag(_("Cancel"),
-                       :class     => "btn btn-default",
-                       "ng-click" => "cancelClicked()")
-
-:javascript
-  ManageIQ.angular.app.value('hostAggregateFormId', '#{@host_aggregate.id}');
-  miq_bootstrap('#form_div');
+= render :partial => "layouts/flash_msg"
+.col-md-12
+  = react('HostAggregateForm', { :recordId => '#{@host_aggregate.id}' })
```

Comment:
i think it should be @host_aggregate.id.to_s

Reply:
i just tested with `@host_aggregate.id` and it worked with no issue. Do you still recomend `@host_aggregate.id.to_s`?

Reply:
check you console log, it works, but most likely you will have console log error.